src/twisted/names/common.py

```diff
@@ -64,6 +64,12 @@ def exceptionForCode(self, responseCode):
         return self._errormap.get(responseCode, DNSUnknownError)
 
     def query(self, query, timeout=None):
+        """
+        Umbrella query handler function.
```

Comment:
This doesn't really add anything over the docstring from `IResolver`, which is inherited when this method is rendered in the documentation (that's why most of these methods lack docstrings). You can see this here: https://twistedmatrix.com/documents/current/api/twisted.names.common.ResolverBase.html#query